styles/main.css

```diff
@@ -27,7 +28,7 @@ a {
 ------------------------------
 --------------------------- */
 
-#about-page, #illustration-page, #webdesign-page, #internal-page {
+#about-page, #work-page, #internal-page {
     position: relative;
```

Comment:
oh nvm you fixed it. good. for good practice, you shouldn't be using `position: absolute` if absolutely necessary.